consensus/src/network.rs

```diff
@@ -263,6 +263,13 @@ impl NetworkTask {
                 Event::RpcRequest((peer_id, msg, callback)) => match msg {
                     ConsensusMsg::BlockRetrievalRequest(request) => {
                         debug!("Received block retrieval request {}", request);
+                        if request.num_blocks() > MAX_BLOCKS_PER_REQUEST {
```

Comment:
should we use min(request.num_blocks(), MAX_BLOCKS_PER_REQUEST);

Reply:
I'd be concerned if we bump this number up and someone is running an old service, that old service would just reject everyone....

Reply:
we currently only use two fix numbers 1 and 3